src/sidebar/util/groups.js

```diff
@@ -35,10 +35,10 @@ function combineGroups(userGroups, featuredGroups, uri) {
 }
 
 function isScopedToUri(group, uri) {
-  // If the group has scope info, the scoping is enforced,
-  // and the uri patterns don't include this page's uri
-  // the group is not selectable, otherwise it is.
-  if (group.scopes && group.scopes.enforced) {
+  // If the group has scope info and the uri patterns
+  // don't include this page's uri the group is not
+  // selectable, otherwise it is.
```

Comment:
It is still the case that the group is selectable if the URI patterns don't match, if scoping is not enforced. I don't see anything in this comment which isn't obvious from the code below any more, so perhaps just remove it.

Reply:
Can you remind me what the reason for removing the `scopes.enforced` check here is?

Reply:
Ah sorry I should have rephrased that last bit but point taken on the "this is a repeat of what the code is doing". isScopedToUri previously was the same as selectable, now it is simply whether the group is within scope. The enforced check was removed since we need to be able to distinguish groups that are in scope from groups that are out of scope but not enforced. Groups that are out of scope but not enforced when we link to them should show up in "Currently Viewing" and groups that are in scope should show up in "Featured Groups".